scintilla/gtk/ScintillaGTK.cxx

```diff
@@ -3250,6 +3250,7 @@ void scintilla_release_resources(void) {
 static void *copy_(void *src) { return src; }
 static void free_(void *doc) { }
 
+GEANY_API_SYMBOL
 GType scnotification_get_type(void) {
 	static gsize type_id = 0;
```

Comment:
Am 23. Juni 2016 05:07:10 MESZ, schrieb Matthew Brush <notifications@github.com>:
>> @@ -3250,6 +3250,7 @@ void scintilla_release_resources(void) {
>>  static void *copy_(void *src) { return src; }
>>  static void free_(void *doc) { }
>>  
>> +GEANY_API_SYMBOL
>>  GType scnotification_get_type(void) {
>>  	static gsize type_id = 0;
>
>Yeah, that's what I meant by "or use gsize without the casts", though I
>think it's kind of weird to intentionally use the (semantically) wrong
>type name for no reason.
>
>---
>You are receiving this because you were mentioned.
>Reply to this email directly or view it on GitHub:
>https://github.com/geany/geany/pull/1038/files/6b5a47d05f575b321ff158044420864ee9e59889#r68169864

g_once_init_leave() takes a gsize which is stored at the value location. That suggested that the value location should point to an gsize compatible location. This guided me.

Anyway the discussion  out of scope for this PR and really minor anyway.